docker/README.md

```diff
@@ -0,0 +1,25 @@
+## Development environment using Docker
```

Comment:
Should we add this info at root `README.md` so it's easier to find? If not should we link from there?

Reply:
Yes of course. I had not added it because I wanted to keep it separate for the review.